src/main.ts

```diff
@@ -37,6 +37,13 @@ export default function (): void {
 		'modifyAddSiteObjectBeforeCreation',
 		(site: Site, newSiteInfo) => {
 			if (newSiteInfo?.customOptions?.useAtlasFramework === 'on') {
+				LocalMain.sendIPCEvent(IPC_EVENTS.TRACK_EVENT, 'v2_site_new_added_atlas', {
+					environment: site.environment,
+					multiSite: site.multiSite,
+					isAtlasBlueprint: !!newSiteInfo.customOptions.atlasUrl,
+					siteServices: site.services,
+					blueprint: newSiteInfo.customOptions.bpId,
```

Comment:
Nit, but is there some disconnect between `blueprint` and `isAtlasBlueprint` here? Could they reference the same thing?

Reply:
I thought it might be easier for Austin to query, but you're right we can probably either populate the `blueprint` or set that value to false